pkgs/by-name/on/onnxruntime/package.nix

```diff
@@ -195,7 +195,7 @@ effectiveStdenv.mkDerivation rec {
     (lib.cmakeFeature "ONNX_CUSTOM_PROTOC_EXECUTABLE" (lib.getExe protobuf))
     (lib.cmakeBool "onnxruntime_BUILD_SHARED_LIB" true)
     (lib.cmakeBool "onnxruntime_BUILD_UNIT_TESTS" doCheck)
-    (lib.cmakeBool "onnxruntime_USE_FULL_PROTOBUF" false)
+    (lib.cmakeBool "onnxruntime_USE_FULL_PROTOBUF" true)
```

Comment:
This is what breaks the [`test_onnx.py`](https://github.com/tinygrad/tinygrad/blob/v0.11.0/test/models/test_onnx.py) of `tinygrad`.

Reply:
Revert it and see if it breaks frigate.

Reply:
It does. :face_holding_back_tears: